src/Layout/Centered.php

```diff
@@ -15,14 +15,23 @@ class Centered extends Generic
 
     public $defaultTemplate = 'layout/centered.html';
 
-    // Default atk4 logo
-    public $image = 'https://github.com/atk4/ui/raw/07208a0af84109f0d6e3553e242720d8aeedb784/public/logo.png';
+    /**
+     * @see \atk4\ui\App::$cdn
+     *
+     * @var null|string
+     */
+    public $image = null;
     public $image_alt = 'Logo';
 
     public function init()
     {
         parent::init();
 
+        // If image is still unset load it when layout is initialized from the App
+        if ($this->image === null && $this->app && $this->app->cdn['atk-logo']) {
+            $this->image = $this->app->cdn['atk-logo'];
```

Comment:
This->app->cdn[‘atk’].’logo.gif’

Reply:
Dark no need to duplicate atk

Reply:
Oh right https://cdn.jsdelivr.net/gh/atk4/ui@1.7.1/public/logo.png will work too. I didn't thought about that.
Then this should simply be:
1. remove atk-logo from App->cdn
2. and here do like this:
```
if ($this->image === null && $this->app) {
    $this->image = $this->app->cdn['atk'] . '/logo.png';
}
```

Reply:
This way is rather bad though.
For example, right now I can use the code that's implemented to just add my image directly to the App and forget about it.
The way you proposed it will never work, unless my image is named "logo.png", which is probably not ideal.

I'd ask you to reconsider, maybe change the cdn string to 'logo' instead of 'atk-logo'

Reply:
@romaninsh and @DarkSide666 how about this solution? I think it's the best of both worlds https://github.com/atk4/ui/pull/750/commits/35ab1b488a17660da558aa0be829b9366c89fb15

Reply:
I've also removed the logo url completely from App later, but forgot in that commit, but you can look over the next changes obviously.